hadoop-tools/hadoop-aws/pom.xml

```diff
@@ -487,6 +487,7 @@
     <dependency>
       <groupId>software.amazon.s3.analyticsaccelerator</groupId>
       <artifactId>analyticsaccelerator-s3</artifactId>
+      <version>1.0.0</version>
```

Comment:
better create a property name for this. Also why don't I see older version here in the diff?

Reply:
thanks @mukund-thakur.. I'd updated the version in the wrong place! updated in hadoop-project, which also has a property name already